docs/src/pages/guides/routing/routing.md

```diff
@@ -63,6 +63,10 @@ You can learn more about this prop in the [composition guide](/guides/compositio
 Here are a few demos with [react-router](https://github.com/ReactTraining/react-router).
 You can apply the same strategy with all the components: BottomNavigation, Card, etc.
 
+**Note**: When the `component` prop is used on the `ButtonBase` and not indicators are
```

Comment:
What are these "indicators"?

Reply:
Maybe I can change it like this?
 


 **Note**: When the `component` prop is used on the `ButtonBase` and no value for the prop `role` is provided then, `role="button"` is provided by the `ButtonBase` to the component by default.
You need to override this attribute if you are rendering a link, e.g. with `role={undefined}`.

Reply:
The indicator would be the href or to props.

Reply:
@eps1lon is it fine now?
I have updated the changes you asked.